util/model_output_parser.py

```diff
@@ -143,7 +160,9 @@ def dataframe_from_ncfile(
     if time_dim is not None:
         time = xarray_ds[time_dim].values
     else:
-        time = [0]
+        # Derive a pseudo time from filename. This is required to process multiple
+        # files of the same file type if the file type has not time dimension.
+        time = [numbers(filename)]
```

Comment:
We're  relying on the filename here. Is there really the need for that?

Reply:
I needed some unique reproducible ID here. Getting it from the filename (without) path seemed the best solution to me.

The problem here is, that I have to process and concatenate multiple files that don't have a time dimension in the dataset. However, the different files differ by the time they represent. So I needed some number that is different for each file but it must be the same every time probtest tests the same file. I think the output files of a model run, that are processed here, should be constant if the model output is reproducible. The whole probtest relies on file names.

It might be that the member id is included in some file names while generating the tolerances. However, the time vector is replace in probtest by 0 to N before writing the csv files. Thus the time is in the end only used to order data from different files among the files.